content/faq/libraries-frameworks/SOAP.mdx

```diff
@@ -0,0 +1,3 @@
+## SoapCore
+
+- DigDes/SoapCore(https://github.com/DigDes/SoapCore)
```

Comment:
Could you please update your PR as following?
- Use our naming convent for a file name as low case kebab style.  Therefore, please change a file name to `soap.md`. 
- This content does not need mdx, so please prefer to use `.md` file.
- Please add front matter at the top of soap.md file as following:

```
---
title: SOAP
showMetadata: true
editable: true
showToc: true
---

```

Reply:
Done. krub

Reply:
Thanks for prompt reply.